tests/recipes/test_lora_dpo_single_device.py

```diff
@@ -73,6 +73,8 @@ def test_training_state_on_resume(
         tune run lora_dpo_single_device \
             --config llama2/7B_lora_dpo_single_device \
             output_dir={tmpdir} \
+            model.lora_attn_modules=['q_proj','v_proj'] \
```

Comment:
I think the better way to do this would be to just modify the source of truth test model definition here: https://github.com/pytorch/torchtune/blob/3f15030de0ada38626cf38b46fdb310171cd67f1/tests/recipes/utils.py#L205-L227